adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Microsoft.IdentityModel.Clients.ActiveDirectory.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="MSBuild.Sdk.Extras/1.5.4">
```

Comment:
These changes require VS 2017 version 15.6+ according to the docs (https://github.com/onovotny/MSBuildSdkExtras), so we should probably add this to the docs.

Reply:
aboslutely. I can't get the global.json to work correctly from the commandline.... thus will revert my changes back to not include the global.json but go with what is in the current PR.